GitUI/Editor/FileViewerInternal.cs

```diff
@@ -25,8 +25,6 @@ public partial class FileViewerInternal : GitExtensionsControl, IFileViewer
 
         private DiffHighlightService _diffHighlightService = DiffHighlightService.Instance;
 
-        private ViewPosition _previousViewPosition;
```

Comment:
I added this because the FileViewer is cleared on update, i.e. `SetText` is called with empty text inbetween.